src/Services/Processor/Substrate/Events/Implementations/MultiTokens/Approved.php

```diff
@@ -31,7 +31,7 @@ public function run(): void
         $operator = $this->firstOrStoreAccount($this->event->operator);
         $owner =  $this->firstOrStoreAccount($this->event->owner);
 
-        if (is_null($this->event->tokenId)) {
+        if (empty($this->event->tokenId)) {
```

Comment:
We are allowing tokenId with 0 right?

Reply:
Yes the issue was there is a place the tokenId is being default to '' and not null so is_null was not working but yeah gotta allow 0 too, let me revert this and change the defaulting option...